{{cookiecutter.repostory_name}}/app/Dockerfile

```diff
@@ -1,19 +1,38 @@
-FROM python:3.6
+{% if cookiecutter.use_alpine_linux == "y" -%}
+ARG BASE_IMAGE=python:3.6-alpine
+{% else -%}
+ARG BASE_IMAGE=python:3.6
+{% endif -%}
 
-COPY ./bin/s6-overlay-amd64.tar.gz /tmp/s6-overlay-amd64.tar.gz
+FROM $BASE_IMAGE AS base-image
 
-RUN tar xvfz /tmp/s6-overlay-amd64.tar.gz -C / \
-    && rm -f /tmp/s6-overlay-amd64.tar.gz {% if cookiecutter.use_celery == "y" %}\
-    && adduser --disabled-password --gecos '' celery{% endif %}
+{% if cookiecutter.use_alpine_linux == "y" -%}
+RUN apk add postgresql-dev gcc python3-dev musl-dev
+{% endif -%}
 
-COPY ./src/ /srv/app/
+ENV PYTHONUNBUFFERED=1
+ENV PATH=/root/.local/bin:$PATH
 
-WORKDIR /srv/app
+RUN pip3 install --user --no-cache-dir psycopg2-binary==2.8.3 gunicorn==19.9.0
 
-RUN pip3 install --no-cache-dir -r requirements.txt gunicorn\
-    && python3 -m compileall -b .
+WORKDIR /root/app
 
-ENTRYPOINT ["/init"]
+COPY ./src/requirements.txt /root/app/requirements.txt
```

Comment:
why the custom path, different directory structure in docker than in repo? couldn't we just use `COPY ./src/requirements.txt src/requirements.txt` and install from there?

Reply:
Originally it was `COPY ./src/ /srv/app/` . It's not a different directory structure. It's just renaming `src` to `app`. I kept the convention, but I can change it to `src`.

Reply:
lets keep it as simple as we can